tests/AuthenticationTest.php

```diff
@@ -85,6 +95,19 @@ public function testHeaderWithMissingFieldOrQueryAuth()
         );
     }
 
+    public function testHeaderWithEmptyFieldOrQueryAuth()
```

Comment:
I'd recommend adding another test case for `and` auth for having one of the auth param's value as empty and then asserting the exception. This will enforce the required behavior for `and` auth.

Reply:
done